ciphey/basemods/Decoders/galactic.py

```diff
@@ -0,0 +1,98 @@
+# community
+# by https://github.com/lukasgabriel
+
+from typing import Optional, Dict, List
+
+from ciphey.iface import Config, ParamSpec, T, U, Decoder, registry
+
+
+@registry.register
+class Galactic(Decoder[str, str]):
+    def decode(self, ctext: T) -> Optional[U]:
+        """
+        Takes a string written in the 'Standard Galactic Alphabet' 
+        (aka Minecraft Enchanting Table Symbols) and translates it to ASCII text.
+        """
+
+        result = ""
+        galactic_letters = [
```

Comment:
Can you move this into Cipheydists?
https://github.com/Ciphey/CipheyDists
This is our repo for storing alphabets etc :) 
Specifically move it here:
https://github.com/Ciphey/CipheyDists/tree/master/cipheydists/translate (I think?)

And then, to get this into the code:

```
     self.MORSE_CODE_DICT = config.get_resource(
            self._params()["dict"], ciphey.iface.WordList
        )
```

This is from Morse_code :)

It's basically a resource locator / fetcher, I should add this to the docs too :)